wrappers/MathCAD/README.rst

```diff
@@ -1,20 +1,49 @@
-Wrapper of CoolProp for MathCAD 15
+CoolProp Wrapper for MathCAD 15
 ==================================
 
-copyright Scott Polak and Ian Bell, 2013
+| Copyright Scott Polak and Ian Bell, 2013. 
+| Updated by Jeff Henning, 2016.
 
-To Use
-======
+Prerequisites
+==============
+
+* You will need to have Visual Studio 2008 installed (Express version is fine).  Alternatively newer versions of Microsoft Visual Studio C++ should be fine.
```

Comment:
Or you could install Visual Studio 2013+ express, which includes 64-bit compilation.  I use/like Visual Studio 2015 Express.

So, MathCAD15 is 32-bit?